lib/foundation_rails_helper/form_builder.rb

```diff
@@ -129,6 +129,35 @@ def custom_label(attribute, text, options, &block)
       label(attribute, text, options)
     end
 
+    def tag_from_options(name,options)
+        if options and options[:size].present? and options[:value].present?
```

Comment:
Use `&&` instead of `and`.  In Ruby, `and` is more of a short circuit, than making sure all conditions are true.  See http://stackoverflow.com/questions/1426826/difference-between-and-and-in-ruby for a reference.